OpenEdXMobile/src/main/java/org/edx/mobile/http/cache/CacheManager.java

```diff
@@ -12,71 +12,99 @@
 import org.edx.mobile.util.Sha1Util;
 
 import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
 import java.io.IOException;
 import java.nio.charset.Charset;
 
+/**
+ * The cache manager for HTTP responses. The cache is stored on the filesystem, within the
+ * application directory.
+ *
+ * @deprecated This is deprecated since with the transition to OkHttp, we're now relying on it's own
```

Comment:
The `@deprecated` annotation implies a "TODO" by itself, so we shouldn't need to add it as a boilerplate.